client/components/UserSettings/index.jsx

```diff
@@ -83,53 +83,57 @@ class UserSettings extends Component {
     }
 
     render() {
-        const { ats, isLoggedIn, username, email } = this.props;
-        const contents = isLoggedIn ? (
-            <Row data-test="user-settings-authorized">
-                <Col>
-                    <h2>USER DETAILS</h2>
-                    <p>{username}</p>
-                    <p>{email}</p>
-                </Col>
-                <Col>
-                    <h2>Assistive Technology</h2>
-                    <p>Add the assistive technologies that you can use</p>
-                    <Form onSubmit={this.onSubmit}>
-                        <Form.Group controlId="formBasicCheckbox">
-                            {ats &&
-                                ats.map(at => {
-                                    return (
-                                        <Form.Check
-                                            id={at.name}
-                                            key={at.id}
-                                            label={at.name}
-                                            onChange={this.onCheckboxClicked}
-                                            checked={
-                                                this.state[at.name]
-                                                    ? this.state[at.name]
-                                                          .checked
-                                                    : false
-                                            }
-                                        />
-                                    );
-                                })}
-                        </Form.Group>
-                        <Button variant="primary" type="submit">
-                            Save
-                        </Button>
-                    </Form>
-                </Col>
-            </Row>
-        ) : (
-            <p data-test="user-settings-unauthorized">Unauthorized</p>
-        );
+        const { ats, isLoggedIn, username, email, loadedUserData } = this.props;
+        const contents =
+            loadedUserData && isLoggedIn ? (
+                <Row data-test="user-settings-authorized">
+                    <Col>
+                        <h2>USER DETAILS</h2>
+                        <p>{username}</p>
+                        <p>{email}</p>
+                    </Col>
+                    <Col>
+                        <h2>Assistive Technology</h2>
+                        <p>Add the assistive technologies that you can use</p>
+                        <Form onSubmit={this.onSubmit}>
+                            <Form.Group controlId="formBasicCheckbox">
+                                {ats &&
+                                    ats.map(at => {
```

Comment:
I missed this before. If you do `const { ats = [], isLoggedIn, username, email, loadedUserData } = this.props;`, then you don't need to do this check